### ISSUE TITLE
Can't get kanboard to work with Mariadb

### ISSUE THREAD

**Issue by sjlovell** [fguillot/kanboard#1288]
Hi, I've successfully got Kanboard installed using SQLLite and kanboard worked great, but I am now trying to get it running against MySQL/Mariadb. I've followed the instructions:
I've installed pdo_sql and created a database named kanboard.

I've changed  config.php file to have the correct driver, username, password, hostname and dbname.

But when I browse to localhost/kanboard in chrome it does nothing. Blank white screen.

My server version is 5.5.5-10.0.20 MariaDB.

The kanboard database is empty - no tables. Should I be running a script to create them? Where can I find some output / logging to determine what's happening?

**Comment by BlueTeck** [comment]
You have to enable the debugging: https://github.com/fguillot/kanboard/blob/master/doc/config.markdown#enabledisable-debug-mode

**Comment by sjlovell** [comment]
Thanks.

Found the problem.
I'm not a Linux or mysql user until installing this, so if something isn't in the documentation it throws me.

The missing step was as well as creating the database I needed to create the user and give it correct rights - as detailed in 
http://stackoverflow.com/questions/1559955/host-xxx-xx-xxx-xxx-is-not-allowed-to-connect-to-this-mysql-server

So essentially:

CREATE USER 'kanboarduser'@'localhost' IDENTIFIED BY 'password';

GRANT ALL PRIVILEGES ON *.* TO 'kanboarduser'@'localhost' WITH GRANT OPTION;

CREATE USER 'kanboarduser'@'%' IDENTIFIED BY 'password';

GRANT ALL PRIVILEGES ON *.* TO 'kanboarduser'@'%' WITH GRANT OPTION;

FLUSH PRIVILEGES;
